src/node.cc

```diff
@@ -3637,6 +3637,8 @@ static void ParseArgs(int* argc,
   const char** new_v8_argv = new const char*[nargs];
   const char** new_argv = new const char*[nargs];
   const char** local_preload_modules = new const char*[nargs];
+  bool use_bundled_ca = false;
+  bool use_openssl_ca = false;
```

Comment:
This seems more opaque, would this be enough of a speed increase to warrant the loss of clarity?

cc/ @sam-github from https://github.com/nodejs/node/pull/12087#discussion_r109003084